src/App.js

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
+import BooksList from './components/BooksList';
+import BooksForm from './components/BooksForm';
 
 function App() {
   return (
     <div className="App">
-      <h1> Book Store Project...</h1>
+      <BooksList />
+      <BooksForm />
     </div>
   );
 }
```

Comment:
- relocate this file into `"src/components"` folder
- it should accept book as a prop [[see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Book/AccceptBookObjectAsProp.png)]

Reply:
Hi Shey 
This seems to be in the Book Component :
Implement the Book component
It should accept a book object as a prop.
It should represent a single row in the Books table.

https://github.com/microverseinc/project-redux-bookstore/blob/master/milestones/MILESTONE_2.md

Reply:
i dont know whats wrong with me today wth BOOK component 😃
(ignore my comment please, correct the rest )